exercises/simple-cipher/src/test/java/Step3IncorrectKeyCipherTest.java

```diff
@@ -0,0 +1,48 @@
+package java;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
This import is unused and is causing the build to fail. `Unused import - org.junit.Assert.assertEquals.`